crates/red_knot_python_semantic/src/types/call/bind.rs

```diff
@@ -1200,8 +1208,9 @@ impl<'db> BindingError<'db> {
                         .nth(parameter_index)
                         .map(|param| param.range())
                         .unwrap_or(func_def.parameters.range);
-                    span = span.with_range(range);
-                    Some(span)
+                    let name_span = span.clone().with_range(func_def.name.range);
+                    let parameter_span = span.with_range(range);
+                    Some((name_span, parameter_span))
                 } else {
```

Comment:
TODO for future: we can also handle `BoundMethod` and maybe some other types here